sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -555,4 +556,360 @@ public void beginRecognizeReceiptsWithOptions() throws IOException {
+                            jobTitlesItems.stream().forEach(formField -> {
+                                if (FieldValueType.STRING == formField.getValue().getValueType()) {
+                                    String jobTitle = formField.getValue().asString();
+                                    System.out.printf("Job Title: %s, confidence: %.2f%n",
+                                        jobTitle, jobTitles.getConfidence());
+                                }
+                            });
+                        }
+                    }
+                    FormField addresses = recognizedFields.get("Addresses");
+                    if (addresses != null) {
+                        if (FieldValueType.LIST == addresses.getValue().getValueType()) {
+                            List<FormField> addressesItems = addresses.getValue().asList();
+                            addressesItems.stream().forEach(formField -> {
+                                if (FieldValueType.STRING == formField.getValue().getValueType()) {
+                                    String address = formField.getValue().asString();
+                                    System.out.printf("Address: %s, confidence: %.2f%n", address,
+                                        addresses.getConfidence());
+                                }
+                            });
+                        }
+                    }
+                    FormField companyName = recognizedFields.get("CompanyNames");
+                    if (companyName != null) {
+                        if (FieldValueType.LIST == companyName.getValue().getValueType()) {
+                            List<FormField> companyNameItems = companyName.getValue().asList();
+                            companyNameItems.stream().forEach(formField -> {
+                                if (FieldValueType.STRING == formField.getValue().getValueType()) {
+                                    String companyNameValue = formField.getValue().asString();
+                                    System.out.printf("Company name: %s, confidence: %.2f%n", companyNameValue,
+                                        companyName.getConfidence());
+                                }
+                            });
+                        }
+                    }
+                }
+            });
+        // END: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeBusinessCardsFromUrl#string-RecognizeBusinessCardsOptions
+    }
+
+    /**
+     * Code snippet for {@link FormRecognizerAsyncClient#beginRecognizeBusinessCards(Flux, long)}
+     *
+     * @throws IOException Exception thrown when there is an error in reading all the bytes from the File.
+     */
+    public void beginRecognizeBusinessCards() throws IOException {
+        // BEGIN: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeBusinessCards#Flux-long
+        File businessCard = new File("{file_source_url}");
```

Comment:
shouldn't be an url. Maybe something like what you have on receipt `local/file_path/fileName.jpg` ?

Reply:
nice catch